modules/claims_api/app/controllers/claims_api/v2/blueprints/meta_blueprint.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module V2
+    module Blueprints
+      class MetaBlueprint < Blueprinter::Base
+        association :data, blueprint: AutoEstablishedClaimBlueprint do |claim, _options|
+          claim
+        end
+
+        field :meta, if: ->(_field_name, claim, _options) { !claim.transaction_id.nil? } do |claim, _options|
```

Comment:
`!` of `nil?` ?

Reply:
@FonzMP I'm not sure I did that, I need to test but I think Rubocop made that change.  Not totally sure of that but I will need to test tomorrow.